index.css

```diff
@@ -0,0 +1,231 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.d-grid {
+  display: grid;
+  grid-template-columns: auto;
+  grid-gap: 1rem;
+}
+
+.headericons {
+  width: 100%;
+}
+
+.margin {
+  margin: 30px;
+}
+
+.recent {
+  font-size: 30px;
+  font-family: "Crete Round", serif;
+  text-align: center;
+  align-items: center;
+}
+
+.container {
+  background-color: white;
+  padding: 1rem;
+}
```

Comment:
I think it would be better if you use `padding: 120px 24px; ` to match match the Figma design. 

Your Design

![image](https://user-images.githubusercontent.com/83285992/143874120-a18c3aae-7261-4e75-b0c4-a9b62269aac6.png)


Figma Design
![image](https://user-images.githubusercontent.com/83285992/143873911-5425f552-92a7-4b73-8912-2ad778ef590b.png)